packages/koenig-lexical/test/e2e/cards/call-to-action-card.test.js

```diff
@@ -361,6 +361,19 @@ test.describe('Call To Action Card', async () => {
         await expect(imgLocator).not.toBeVisible();
     });
 
+    test('has a progress bar while uploading', async function () {
+        const filePath = path.relative(process.cwd(), __dirname + `/../fixtures/large-image.jpeg`);
+    
+        await focusEditor(page);
+        await insertCard(page, {cardName: 'call-to-action'});
+        const fileChooserPromise = page.waitForEvent('filechooser');
+        await page.click('[data-testid="media-upload-placeholder"]');
+        const fileChooser = await fileChooserPromise;
+        await fileChooser.setFiles([filePath]);
+        const progressBar = page.locator('[data-testid="progress-bar"]');
+        await expect(progressBar).toBeVisible();
```

Comment:
Is it worth expanding this test a bit to make sure the progress bar correctly disappears too once the upload is finished?